jina/main/parser.py

```diff
@@ -75,6 +75,8 @@ def set_hub_new_parser(parser=None):
                         help='where to output the generated project dir into.')
     parser.add_argument('--template', type=str, default='https://github.com/jina-ai/cookiecutter-jina-hub.git',
                         help='s directory containing a project template directory, or a URL to a git repository.')
+    parser.add_argument('--type', type=str, default='pod',
```

Comment:
```suggestion
    parser.add_argument('--type', type=str, default='pod',  choices=['pod', 'app', 'template'],
```

use `choices` for better semantic

Reply:
done